src/bin/electrs.rs

```diff
@@ -151,9 +158,32 @@ fn run_server(config: Arc<Config>) -> Result<()> {
     Ok(())
 }
 
+fn generate_salt() -> String {
```

Comment:
it might be helpful to give this function a name closer to its intended use -- related to anonymizing rpc logging? or add a comment